### PR TITLE
Remove fancy tag input

### PR DESCRIPTION
**Pull request by charmander**
It looks nice, at the cost of:

- ability to copy
- ability to edit tags after they’re created
- browser undo and redo
- probably more

Clients already provide whole-word selection/navigation/deletion, so nobody’s missing out on much by not having the “x” buttons anymore.

Also, 14px text instead of 13⅓px, because it felt right. I suspect that every field should be at least this size.

### CODE REVIEW

File: weasyl/templates/submit/character.html [hunk 1]
```diff
@@ -57,7 +57,7 @@ <h3>Description</h3>
 
     <div class="column tagging">
         <h3>Tags</h3>
-        <textarea name="tags" class="input tags-textarea" rows="3"></textarea>
+        <textarea name="tags" class="input tags-textarea" rows="3" required></textarea>
```
Comment (kfkitsune) [inline weasyl/templates/submit/character.html:60]: See comment on ``required`` for the modification to the journal code; I don't think we actually enforce this anywhere.

File: weasyl/templates/submit/multimedia.html [hunk 2]
```diff
@@ -72,7 +72,7 @@ <h3>Description</h3>
 
     <div class="column tagging">
       <h3>Tags</h3>
-      <textarea name="tags" class="input tags-textarea" rows="3"></textarea>
+      <textarea name="tags" class="input tags-textarea" rows="3" required></textarea>
```
Comment (kfkitsune) [inline weasyl/templates/submit/multimedia.html:75]: Likewise, no Python enforcement here. See other comments.

**Review by kfkitsune — COMMENTED** [review]
A few observations, but functionally it looks okay. Unless I am missing something, it does change the behavior for requiring tags for journal/character/multimedia submissions, though.

File: weasyl/templates/submit/journal.html [hunk 3]
```diff
@@ -27,7 +27,7 @@ <h3>Content</h3>
 
     <div class="column tagging">
       <h3>Tags</h3>
-      <textarea name="tags" class="input tags-textarea" rows="3"></textarea>
+      <textarea name="tags" class="input tags-textarea" rows="3" required></textarea>
```
[the remark this answers is not in the document]
Comment (charmander) [reply weasyl/templates/submit/journal.html:30]: Eep, this one was accidental. Thanks!

File: weasyl/templates/submit/multimedia.html (hunk 2 above)
Comment (charmander) [reply weasyl/templates/submit/multimedia.html:75]: I think we enforce this? Or the code seems to be trying to, at least. https://github.com/Weasyl/weasyl/blob/25699ec20c7fc62a8433524733ba2cd2d20e4674/weasyl/submission.py#L97-L98

Comment (charmander) [reply weasyl/templates/submit/multimedia.html:75]: Yep, checked; I get the error for new multimedia submissions on master.

**Review by kfkitsune — APPROVED** [review]
LGTM. It functioned okay prior to the change removing the ``required`` attribute from the journal/character tag fields to match current functionality, so this specific review is a visual review only.

(To be perfectly honest, my eyes kinda slid over the ``@_create_submission(expected_type=3)`` invocation for multimedia, which itself did have the tag quantity check. My bad, there.)

**Pull request merged**